lib/notesclub/searches/remove.ex

```diff
@@ -0,0 +1,20 @@
+defmodule Notesclub.Searches.Remove do
+  require Logger
+
+  alias Notesclub.Searches
+
+  @number_of_days_to_keep_search_results 30
+
+   # public function so we can mock it in tests
+   def number_of_days_to_keep_search_results(),
+    do: @number_of_days_to_keep_search_results
+
+  def eliminate() do
+    {count, nil} = Timex.now()
+    |> Timex.beginning_of_day()
+    |> Timex.shift(days: -@number_of_days_to_keep_search_results)
+    |> Searches.delete_by_date()
+
+    Logger.info("#{count} search records remove.")
```

Comment:
We need to add a result after Logger. Otherwise, we'll it will always return `:ok` and tests would fail.
Also, instead of returning `{count, nil}`, we could return `{:ok, count}`. We don't need to expose how we delete the records internally. But then we'll need to change your tests.